### DISCUSSION TITLE
Error while Setting up Cypress with Next.js

### DISCUSSION THREAD

**Discussion by Sattu2806** [vercel/next.js#59644] [Help]
### Summary

So, I am setting up my nextjs with cypress using this command (npx create-next-app@latest --example with-cypress with-cypress-app) 

and I am getting something like this ----------------

<img width="1440" alt="Screenshot 2023-12-15 at 12 41 33 PM" src="https://github.com/vercel/next.js/assets/127772255/409183e6-5b2a-402d-8161-cb4ce75daacf">

Creating a new Next.js app in /Users/satendermaurya/Documents/Sattu/MERN/NextJs Blog/with-cypress-app.

Downloading files for example with-cypress. This might take a moment.

Installing packages. This might take a couple of minutes.

npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "^latest" of package "next@^latest": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in: /Users/satendermaurya/.npm/_logs/2023-12-15T07_11_06_485Z-debug-0.log

Aborting installation.
  npm install has failed.

**Comment by icyJoseph** [comment]
Remove the `latest` tag I guess:

```shell
npx create-next-app --example with-cypress with-cypress-app
```

If I remember correctly, create-next-app will update itself when running, and also, it'll install the latest next.js version in the target project.

**Reply by Sattu2806** [reply]
Getting same error !

<img width="1440" alt="Screenshot 2023-12-15 at 1 58 45 PM" src="https://github.com/vercel/next.js/assets/127772255/e268915b-36c9-465b-8fe6-3045318a31a2">

**Reply by Sattu2806** [reply]
After running it I am getting a folder with name `with-cypress-app` 
<img width="1440" alt="Screenshot 2023-12-15 at 2 01 56 PM" src="https://github.com/vercel/next.js/assets/127772255/c372a9ab-ad08-445f-97ef-9a347e057818">

But It is not working when I do npm run dev

getting this error

<img width="1440" alt="Screenshot 2023-12-15 at 2 03 42 PM" src="https://github.com/vercel/next.js/assets/127772255/764d1729-4329-4372-ad36-8157ad3c8859">

**Reply by icyJoseph** [reply]
ohh I see ,there's an error in that example... someone set the `next` version on the project as `^latest`, please change it to `"next": "latest"`


```json
  "dependencies": {
    "next": "latest",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
```

I am making a PR to fix it, it happened a few days ago, and it is only on that one example:

```shell
rg "\^latest" examples
examples/with-cypress/package.json
13:    "next": "^latest",
```
